src/evaluate/info.py

```diff
@@ -54,6 +77,7 @@ class EvaluationModuleInfo:
     streamable: bool = False
     format: Optional[str] = None
     module_type: str = "metric"  # deprecate this in the future
```

Comment:
Unrelated to this PR, but one suggestion would be to add a deprecation warning so we can alert users when this will be removed / remind ourselves to remove it :)